src/clients/c/tb_client/context.zig

```diff
@@ -179,7 +178,6 @@ pub fn ContextType(
             );
             errdefer context.client.deinit(context.allocator);
 
-            context.messages_available = constants.client_request_queue_max;
```

Comment:
I _think_ there was a subtle bug  here --- there's also `.register` request, which _won't_ be counted by this counter. New impl should be fine (but `.register` is an extra gotcha for the users to be aware of).